### DISCUSSION TITLE
SciPy 2023 Tutorial

### DISCUSSION THREAD

**Discussion by bjlittle** [pyvista/pyvista-tutorial#208] [Q&A] [answered]
@tkoyama010 @banesullivan @akaszynski Do you have a link to the NumFocus YouTube recording of you guys teaching the tutorial at SciPy 2023?

Thanks 👍

**Comment by tkoyama010** [comment, marked as answer]
Yes, there is. However, the film crew were unable to arrive due to a storm on the day of the event, so we filmed ourselves. Therefore, it does not resemble well in the filming.
https://youtu.be/lxgqlNlqR9c?si=1Uoi3iM79YK7NXA7

**Reply by banesullivan** [reply]
I don't think the screen capture in that video is from our tutorial at all. Something got mixed up

**Reply by bjlittle** [reply]
> I don't think the screen capture in that video is from our tutorial at all. Something got mixed up

Ahh okay, that explains it. I wondered what happened.

Thanks @tkoyama010 and @banesullivan 🍻
